### ISSUE TITLE
[E2] Orden de las cartas en la mano del Superstar

### ISSUE THREAD

**Issue by xenon10gh** [RodrigoToroIcarte/IIC2113-2023-1#147]
Hola, tenia una duda en la forma que le tengo que ordenar la lista de la mano de la superstar para el metodo ShowCars(...). 

En el enunciado salen que la mano de este arsenal por ejemplo es la siguiente en ese orden:
![image](https://github.com/RodrigoToroIcarte/IIC2113-2023-1/assets/105937232/3d9affdb-9efb-4136-b86f-b33dccd0dcb4)

En el primer testcase de NoEffects sale la siguiente mano:
![image](https://github.com/RodrigoToroIcarte/IIC2113-2023-1/assets/105937232/c2daea63-1c0e-433c-b7c8-2ddb61b78f6b)

Y en el metodo ShowCards(...) se imprime 
![image](https://github.com/RodrigoToroIcarte/IIC2113-2023-1/assets/105937232/e47d2d9a-ea2e-467a-9352-c2fc612baf6a)
Donde se agrega *Big Boot* ya que se robo al inicio del turno. Claramente se ve que esta al reves que en la imagen anterior.

**Comment by RodrigoToroIcarte** [comment]
Sí, son dos cosas distintas. Esa parte del enunciado solo discute las cartas que Kane tendrá en su mano, haciendo referencia a que las últimas cartas son las que están al tope del arsenal. No se refiere al orden en que deben aparecer en la mano de Kane.

Justo abajo de eso se discute el orden en que se deben ir agregando cartas a la mano del jugador. La carta robada se debe insertar al final de la mano. Por eso la última carta que roba Kane siempre termina al final de su mano.
 
![Screenshot from 2023-09-06 21-02-40](https://github.com/RodrigoToroIcarte/IIC2113-2023-1/assets/7000518/0197168f-bdb0-4659-93e5-67879b19e3ff)

También podría haber sido consistente con el orden de las cartas en el ejemplo que mencionas. Pero lo dejé así porque era más fácil visualizar que las 7 últimas cartas son las mismas que terminan en la mano de Kane.
